### DISCUSSION TITLE
Icon request: full trash bin

### DISCUSSION THREAD

**Discussion by odysseuscm** [FortAwesome/Font-Awesome#21129] [Icon Requests]
I'd appreciate an icon that shows the trash bin isn't empty; e.g. with the lid being lifted a bit on one side.
Thanks for considering!

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
